python/cugraph/ktruss/ktruss_subgraph_wrapper.pyx

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cugraph.ktruss.ktruss_subgraph cimport *
+from cugraph.structure.c_graph cimport *
+from cugraph.utilities.column_utils cimport *
+from libcpp cimport bool
+from libc.stdint cimport uintptr_t
+from libc.stdlib cimport calloc, malloc, free
+from libc.float cimport FLT_MAX_EXP
+
+import cudf
+import cudf._lib as libcudf
+import rmm
+import numpy as np
+
+
+def ktruss_subgraph(graph_ptr,k, subgraph_truss):
+    """
+    Call gdf_katz_centrality
+    """
+    cdef uintptr_t graph = graph_ptr
+    cdef Graph* g = <Graph*>graph
+
+	# cdef int kc = <int>k
+    cdef uintptr_t subgraph = subgraph_truss
+    cdef Graph* gsub = <Graph*>subgraph
+	
+    k_truss_subgraph(g, k, gsub);
+
```

Comment:
if gsub.edgeList is not NULL:
        df = cudf.DataFrame()
        df['src'], df['dst'], vals = graph_wrapper.get_edge_list(subgraph)
        if vals is not None:
            df['val'] = vals
            subgraph_truss.from_cudf_edgelist(df, source='src', target='dst', edge_attr='val')
        else:
            subgraph_truss.from_cudf_edgelist(df, source='src', target='dst')
        if input_graph.edgelist is not None:
            subgraph_truss.renumbered = input_graph.renumbered
            subgraph_truss.edgelist.renumber_map = input_graph.edgelist.renumber_map
    if gsub.adjList is not NULL:
        off, ind, vals = graph_wrapper.get_adj_list(subgraph)
        subgraph_truss.from_cudf_adjlist(off, ind, vals)
    if gsub.transposedAdjList is not NULL:
        off, ind, vals = graph_wrapper.get_transposed_adj_list(subgraph)
        subgraph_truss.transposedadjlist = subgraph_truss.transposedAdjList(off, ind, vals)